### PR TITLE
unitq integration

### PR DESCRIPTION
**Pull request by wahlforss**
### What does this PR do?

Added unitQ integration

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ x] Feature or bugfix has tests
- [x ] Git history is clean
- [ x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by alexvetras** [comment]
@wahlforss 

1. It looks like the PR is failing our validations, because UnitQ still needs to make an entry in the [CODEOWNERS](https://docs.datadoghq.com/developers/marketplace/#code-owners) file. 

2. The integration will need an [out-of-the-box dashboard](https://docs.datadoghq.com/developers/marketplace/#dashboards-and-monitors).

3. I see that your configuration instructions are to: "Contact your customer success manager to enable the unitQ Datadog integration." Unfortunately, I am not sure we can move forward with the integration if customers cannot set it up themselves. Would you be able to provide steps for the customer to enable the integration in place of the contact CSM language?

4. There is one metric listed in your CSV file: "unitq.user_feedback". Is this the only metric being sent in? How many tags do you estimate will be used with this metric?

5. The CHANGELOG is missing a date, could you please add one?

Thanks!

**Comment by wahlforss** [comment]
> @wahlforss
> 
> 1. It looks like the PR is failing our validations, because UnitQ still needs to make an entry in the [CODEOWNERS](https://docs.datadoghq.com/developers/marketplace/#code-owners) file.
> 2. The integration will need an [out-of-the-box dashboard](https://docs.datadoghq.com/developers/marketplace/#dashboards-and-monitors).
> 3. I see that your configuration instructions are to: "Contact your customer success manager to enable the unitQ Datadog integration." Unfortunately, I am not sure we can move forward with the integration if customers cannot set it up themselves. Would you be able to provide steps for the customer to enable the integration in place of the contact CSM language?
> 4. There is one metric listed in your CSV file: "unitq.user_feedback". Is this the only metric being sent in? How many tags do you estimate will be used with this metric?
> 5. The CHANGELOG is missing a date, could you please add one?
> 
> Thanks!

Hi Alex!

Thanks for your feedback. I've implemented your requests. 

The number of tags should not be too many in the end, roughly 20-30.

**Review by jose-manuel-almaza — APPROVED** [review]

File: unitq/README.md
Comment (emarsha94) [inline unitq/README.md:7]: We try to avoid sales related material for the integrations page so I think we could remove this

**Review by emarsha94 — CHANGES_REQUESTED** [review]
We highly encourage branding/logo for dashboards. Could you add branding for the dashboard?

File: unitq/README.md
Comment (hestonhoffman) [inline unitq/README.md:15]:
```suggestion
1. In unitQ, go to **Integrations**.
```

File: unitq/metadata.csv
Comment (hestonhoffman) [inline unitq/metadata.csv:2]:
```suggestion
unitq.user_feedback,count,,,,Customer feedback from unitQ.,0,unitq,user feedback,
```

**Review by hestonhoffman — APPROVED** [review]
Couple of small edits

**Comment by wahlforss** [comment]
I've now updated for the requested changes.

File: unitq/README.md
Comment (wahlforss) [reply unitq/README.md]: Done!

**Comment by wahlforss** [comment]
> We highly encourage branding/logo for dashboards. Could you add branding for the dashboard?

Done

File: unitq/manifest.json
Comment (yzhan289) [inline unitq/manifest.json:24]: Do you have another website for unitq?

Comment (wahlforss) [reply unitq/manifest.json]: Yes updated it. All the other plugins have this URL btw.

File: unitq/metadata.csv
Comment (sarah-witt) [inline unitq/metadata.csv:2]:
```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
unitq.user_feedback,count,,,,Customer feedback from unitQ.,0,unitq,user feedback,
```
Do you mind removing the carriage return? Thanks!

**Review by emarsha94 — APPROVED** [review]

**Comment by wahlforss** [comment]
It should be ready to merge now!

**Comment by sarah-witt** [comment]
Hi @wahlforss, I had one last comment from the agent integrations side- do you mind renaming the changelog to CHANGELOG.md to match the entry in the manifest? https://github.com/DataDog/integrations-extras/pull/1512/files#diff-a773cfac1d87242fc239c0dd93333979db099d28e8ff529eb3cc2d0f01b50841R10

**Comment by wahlforss** [comment]
> Hi @wahlforss, I had one last comment from the agent integrations side- do you mind renaming the changelog to CHANGELOG.md to match the entry in the manifest? https://github.com/DataDog/integrations-extras/pull/1512/files#diff-a773cfac1d87242fc239c0dd93333979db099d28e8ff529eb3cc2d0f01b50841R10

I don't understand what you mean? The file matches the manifest file? Feel free to update yourself.

**Comment by sarah-witt** [comment]
Hi @wahlforss right now the name is `CHANGELOG.MD`, but the extension needs to be updated so it is named `CHANGELOG.md`

**Comment by wahlforss** [comment]
> Hi @wahlforss right now the name is `CHANGELOG.MD`, but the extension needs to be updated so it is named `CHANGELOG.md`. I have updated it accordingly!

done

File: unitq/manifest.json
Comment (emarsha94) [inline unitq/manifest.json:13]: We highly encourage adding images here

Comment (wahlforss) [reply unitq/manifest.json:13]: We plan on doing that in a later release. A number of our customers asked for this integration this week, so we would love to ship it before we start adjusting the details. I see that multiple integrations do not have any media. If possible, we would like to add it first and update the integration later. We are unfortunately already delayed according to our promise to multiple customers.

File: unitq/README.md
Comment (emarsha94) [inline unitq/README.md:5]:
```suggestion
The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog.By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback ensuring a happy customer base
```

Comment (wahlforss) [reply unitq/README.md:5]: We got a previous comment that said you don't allow marketing copy here. So I we previously had the copy describing unitQ but removed it due to a comment.

File: unitq/metadata.csv
Comment (sarah-witt) [inline unitq/metadata.csv:2]:
```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```
I'm still seeing the carriage return, I'm going to try to manually fix it

File: unitq/metadata.csv
Comment (sarah-witt) [inline unitq/metadata.csv:1]:
```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
unitq.user_feedback,count,,,,Customer feedback from unitQ.,0,unitq,user feedback,
```

**Review by sarah-witt — APPROVED** [review]
Looks good from the agent integrations side!

File: unitq/README.md
Comment (anawishnoff) [reply unitq/README.md]: Hi @wahlforss,  just to clarify here - we recommend adding descriptive text about your product without any sales or marketing-forward language (such as phrases like "best in class", or "top rated"). I think adding a sentence here about what UnitQ does as a product would help highlight the value of the integration to possible new customers.

File: unitq/README.md
Comment (anawishnoff) [inline unitq/README.md:5]:
```suggestion
The unitQ Datadog integration allows you to forward metrics from unitQ to Datadog. By delivering unitQ metrics to Datadog, users can leverage Datadog's graphing and alerting capabilities to track user feedback - ensuring a happy customer base.
```

File: unitq/README.md
Comment (anawishnoff) [inline unitq/README.md:11]: A hyperlink to the Integrations page within unitQ would be helpful here.

File: unitq/manifest.json
Comment (anawishnoff) [inline unitq/manifest.json:19]:
```suggestion
      "Category::Metrics"
      "Offering::Integration"
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
Wrapping up this review with some last minor comments.

File: unitq/README.md
Comment (wahlforss) [reply unitq/README.md:11]: Unfortunately, we can't do that at the moment.

File: unitq/README.md
Comment (wahlforss) [reply unitq/README.md]: I see. I'll add one sentence.

**Comment by wahlforss** [comment]
> Wrapping up this review with some last minor comments.

fixed

**Review by anawishnoff — COMMENTED** [review]
This looks good to me, the only remaining item is the lack of images on the tile. Our team won't block on this, but we do recommend adding images to highlight the value of your listing to customers. @wahlforss Do you plan on adding images, or would you like to move forward without them?

**Review by anawishnoff — APPROVED** [review]

**Pull request merged**
